### DISCUSSION TITLE
Github Copilot only suggesting 1 line at a time

### DISCUSSION THREAD

**Discussion by adelin-b** [community/community#20163] [Copilot Conversations]
Copilot only give suggestion line by line while it used to predict whole paragraph before.
Also it recently started to give 0/10 solution way more than before on simple tasks.

Same problem as #17979

Maybe implementing regression testing on tests cases would help finding thoses issues.

**Comment by D7EAD** [comment]
Hi!

Under Settings -> Github Copilot on Github, do you have "_[s]uggestions matching public code_" turned on? Having this disabled will result in a drastically worse code recommendation experience. I am unsure if this is the root of this issue, but it is still worth checking!

**Reply by adelin-b** [reply]
Yes it is enable. 
The prediction change by a big margin on refactor cases, also I have only one or zero solution displayed (it still says "Synthesizing 10/10 solutions (Duplicates hidden)
")

**Reply by tmlewin** [reply]
I am encountering the same thing. The biggest challnge i have with copilot is that the quality and accuracy of the suggestions has taken a nosedive  since it went to production.  comapred to 3 moinths ago, there seem to be a maurate. I hope they fix things on the backend
